packages/studio/src/components/VersionNumber.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from "react";
+import { version } from "../../package.json";
+import { ReactComponent as Info } from "../icons/info.svg";
+import ButtonWithModal, { renderModalFunction } from "./common/ButtonWithModal";
+import ReactModal from "react-modal";
+
+const customReactModalStyles = {
+  content: {
+    top: "50%",
+    left: "50%",
+    right: "50%",
+    bottom: "auto",
+    marginRight: "-95%",
+    maxWidth: "600px",
+    transform: "translate(-50%, -50%)",
+  },
+  overlay: {
+    zIndex: 1000,
+  },
+};
+
+/**
+ * Renders the current version number
+ */
+export default function VersionNumber() {
+  const infoSVG = <Info className="h-7" />;
```

Comment:
can move this out of the body of the component (think I prefer that a little here)